modules/tinymce/src/plugins/quickbars/test/ts/browser/ContentEditableTest.ts

```diff
@@ -57,4 +57,13 @@ describe('browser.tinymce.plugins.quickbars.ContentEditableTest', () => {
     TinySelections.select(editor, 'p span', []);
     await pAssertToolbarNotVisible();
   });
+
+  it('TINY-9190: Toolbar is not shown in the fake caret', async () => {
```

Comment:
We probably need some test cases that include some checking of the ancestors.

Reply:
A test for the ancestor with `data-mce-bogus="all"` is added